src/components/importantDetails/ImportantDetails.vue

```diff
@@ -18,39 +18,39 @@
         <tr>
           <th scope="row">Avarage seconds per block</th>
           <td>~ {{ sharedState.rskConfig.secondsPerBlock }} seconds</td>
-          <td>~ {{ sharedState.ethConfig.secondsPerBlock }} seconds</td>
+          <td>~ {{ sharedState.sideConfig.secondsPerBlock }} seconds</td>
         </tr>
         <tr>
           <th scope="row">Confirmations needed for small amounts</th>
           <td>
             {{ rskConfirmations?.smallAmount }} blocks (~ {{ rskConfirmations?.smallAmountTime }}) +
-            2 minutes voting
+            {{ votingTime }}
           </td>
           <td>
             {{ ethConfirmations?.smallAmount }} blocks (~ {{ ethConfirmations?.smallAmountTime }}) +
-            2 minutes voting
+            {{ votingTime }}
           </td>
         </tr>
         <tr>
           <th scope="row">Confirmations needed for medium amounts</th>
           <td>
             {{ rskConfirmations?.mediumAmount }} blocks (~ {{ rskConfirmations?.mediumAmountTime }})
```

Comment:
rename rskConfirmations  to mainConfirmations

Reply:
sure? because, this chain will always be rsk